locales/en.yml

```diff
@@ -916,7 +916,7 @@ en:
     ERR_038_TIE_OPEN: A tie using '+' must be followed by a note.
     ERR_039_TIE_NO_NOTE: A tie using '+' must be preceded by a note.
     ERR_040_HIERARCHY_INVALID: The order of elements cannot be interpreted (%{value}).
-    ERR_041_NESTING_INVALID: The nesting of opening and closing tags for '%{value}' is invalid.
+    ERR_041_NESTING_INVALID: "'qq' and 'r' must appear outside of the beamed group."
```

Comment:
No, this is not what is meant. This warning appears if you are closing a group before an inside one. Something like `{...(... }...)`. It can also happen with grace notes. So something like `qq...{...r...}`, or the opposite `{...qq...}...r`